### DISCUSSION TITLE
q-value percentile & Protein Inferences

### DISCUSSION THREAD

**Discussion by ClaireS24** [vdemichev/DiaNN#684] [Q&A]
Hello,

I have a question regarding precursor filtering. Is possible in DIA-NN to set q-value percentile, to choose % of runs in which precursors are identified? And if so, is it possible to change q-value percentile to for example 0.1 (precursors identified in 10 % of samples)?

Also, can you please explain the difference in identifications if I change ´Protein inference´? In our case, we tried to run two experiments with the setting ´Isoforms IDs´, and then ´Genes´. In some runs, the ´Isoforms IDs´ had more identification of precursors and in some runs ´Genes´. Can you please clarify why that is?

Thank you and best regards.

Claire

**Comment by vdemichev** [comment]
Hi Claire,

DIA-NN calculates global q-values in a statistically-justified way, i.e. no need for extra filtering based on detection rate to keep FDR controlled. However this can of course make sense for the specific experiment design, then, can do this in R or Python, e.g. using the code Lee posted above.

The main effect is how 'proteotypicity' is defined. There's also very minor effect on protein grouping. This setting cannot affect precursor ID numbers, if it does, please share the logs.

Best,
Vadim
